### PR TITLE
Check for "any" in returned IPs

### PR DESCRIPTION
**Pull request by kozlovic**

### CODE REVIEW

File: server/server.go [hunk 1]
```diff
@@ -927,6 +927,14 @@ func (s *Server) getClientConnectURLs() []string {
 	if err != nil || len(urls) == 0 {
 		// We are here if s.opts.Host is not "0.0.0.0" nor "::", or if for some
 		// reason we could not add any URL in the loop above.
+		if err == nil && ipAddr.IP.IsUnspecified() {
+			// We had a case where a Windows VM was hosed and would have err == nil
+			// and not add any address in the array in the loop above, and we
+			// ended-up returning 0.0.0.0, which is problematic for Windows clients.
+			// If that happens, I think we should panic to let the user know that there
+			// is something wrong...
+			panic(fmt.Errorf("address %q can not be resolved properly", ipAddr.IP.String()))
```
Comment (ColinSullivan1) [inline server/server.go]: Maybe log and ignore this url?  The server was still running and accepting local connections.

**Comment by kozlovic** [comment]
@derekcollison This is ridiculous, if I had put the comments above the `if` statement, the coverall score would be better. It looks like it is counting comments as relevant code...

File: server/server.go (hunk 1 above)
Comment (kozlovic) [inline server/server.go]: That's another approach. What was the other symptoms you had with your VM, how did you detect that there was something wrong and needed to be rebooted?

Comment (kozlovic) [inline server/server.go]: Server could indeed return nothing for the list of client connect URLs, which is fine, it won't break anything.

Comment (ColinSullivan1) [inline server/server.go]: I couldn't reach anything outside the VM.  If it happens again, I'll try to resolve it to test this case.

**Comment by kozlovic** [comment]
Closing, will resubmit with different approach.

**Pull request closed without merging**